### ISSUE TITLE
com.jetbrains.plugins.webDeployment.connections.RemoteConnectionManager.openConnection

### ISSUE THREAD

**Issue by Gamesh** [Haehnchen/idea-php-symfony2-plugin#840]
```
com.jetbrains.plugins.webDeployment.connections.RemoteConnectionManager.openConnection(Lcom/jetbrains/plugins/webDeployment/ConnectionOwner;Ljava/lang/String;Lcom/jetbrains/plugins/webDeployment/config/WebServerConfig;Lcom/jetbrains/plugins/webDeployment/config/FileTransferConfig$Origin;Lcom/intellij/util/EventDispatcher;Lcom/intellij/openapi/progress/ProgressIndicator;)Lcom/jetbrains/plugins/webDeployment/connections/RemoteConnection;
java.lang.NoSuchMethodError: com.jetbrains.plugins.webDeployment.connections.RemoteConnectionManager.openConnection(Lcom/jetbrains/plugins/webDeployment/ConnectionOwner;Ljava/lang/String;Lcom/jetbrains/plugins/webDeployment/config/WebServerConfig;Lcom/jetbrains/plugins/webDeployment/config/FileTransferConfig$Origin;Lcom/intellij/util/EventDispatcher;Lcom/intellij/openapi/progress/ProgressIndicator;)Lcom/jetbrains/plugins/webDeployment/connections/RemoteConnection;
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.utils.RemoteWebServerUtil.collectRemoteFiles(RemoteWebServerUtil.java:71)
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.WebDeploymentProjectComponent$MyTimerTask$1.run(WebDeploymentProjectComponent.java:73)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:710)
	at com.intellij.openapi.progress.impl.CoreProgressManager$3.run(CoreProgressManager.java:179)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:568)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:519)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$1.run(ProgressManagerImpl.java:128)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:309)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```
Plugin v0.12.132
PHPStorm v2016.3.2

occurred about 50 times

coould have something to do with "Download remote files" feature that is enabled.

**Comment by Gamesh** [comment]
Disabling download remote files had no effect

**Comment by Haehnchen** [comment]
fixed by 0.13.133 and #826, but needs validation

**Comment by Gamesh** [comment]
appears to be fixed, i no longer see this error
